drivers/clock_control/clock_control_mcux_scg.c

```diff
@@ -60,9 +61,13 @@ static int mcux_scg_get_rate(const struct device *dev,
 	case KINETIS_SCG_FIRC_CLK:
 		clock_name = kCLOCK_ScgFircClk;
 		break;
+#if !defined(CONFIG_SOC_MKE15Z7)
 	case KINETIS_SCG_SPLL_CLK:
 		clock_name = kCLOCK_ScgSysPllClk;
 		break;
+	case KINETIS_SCG_SPLL_CLK:
```

Comment:
This looks like a duplicate switch case to the one above? Did you indent to have something like the following here:
```
#if !defined(CONFIG_SOC_MKE15Z7)
	case KINETIS_SCG_SPLL_CLK:
		clock_name = kCLOCK_ScgSysPllClk;
		break;
#else
	case KINETIS_SCG_SPLL_CLK:
    clock_name = kCLOCK_ScgLpFllClk;
		break;
#endif
```